tests/test_logger.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+"""Watchmaker main test module."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals, with_statement)
+
+from watchmaker.logger import exception_hook
+from watchmaker.logger import LOG_LEVELS
+from watchmaker.logger import make_log_dir
+from watchmaker.logger import prepare_logging
+
+import logging
+import os
+import shutil
+import sys
+import unittest
```

Comment:
isort order is builtin imports, then third-party imports, then application imports. it's also ordered alphabetically (and case matters). switch this around to:

```
import logging
import os
import shutil
import sys
import unittest

from watchmaker.logger import LOG_LEVELS
from watchmaker.logger import exception_hook
from watchmaker.logger import make_log_dir
from watchmaker.logger import prepare_logging
```